lib/presentation/widgets/logo_widget.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class LogoWidget extends StatelessWidget {
+  const LogoWidget({Key? key}) : super(key: key);
+
+  // TODO: Add localization
+  static const logoText = 'WaterApp';
+  static const logoImagePath = "assets/images/logo.svg";
```

Comment:
you could use https://pub.dev/packages/flutter_gen for assets. Pls, create ticket for this

Reply:
Ok, I am creating a ticket now, but but i will use it later